.jenkins/pipelines/Agnostic/Linux/Jenkinsfile

```diff
@@ -38,6 +38,37 @@ def simulationContainerTest(String version, String build_type, List extra_cmake_
     }
 }
 
+def buildCrossPlatform(String version) {
+    stage("Ubuntu ${version} OP-TEE Build") {
+        node(AGENTS_LABELS["ubuntu-nonsgx"]) {
+            timeout(GLOBAL_TIMEOUT_MINUTES) {
+                cleanWs()
+                checkout scm
+                def os = version == "20.04" ? "focal" : "bionic"
+                def task = """
+                           cd ${WORKSPACE}
+
+                           sdk_path=`pwd`
```

Comment:
You should always use `$()` as backticks are a deprecated method of command substitution. `$()` is also POSIX compatible